packages/flow_parser/flow_parser.0.229.1/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+maintainer: "flow@fb.com"
+authors: ["Flow Team <flow@fb.com>"]
+homepage: "https://github.com/facebook/flow/tree/master/src/parser"
+bug-reports: "https://github.com/facebook/flow/issues"
+license: "MIT"
+
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "ocaml" {>= "4.14.0" & < "5.2"}
+  "dune" {>= "3.2"}
```

Comment:
in the repository, it seems that only Dune >= 3.0 is required:

- https://github.com/facebook/flow/blob/392b6d3008caa40a38627de876f3cd29611f6be4/flow_parser.opam#L13
- https://github.com/facebook/flow/blob/392b6d3008caa40a38627de876f3cd29611f6be4/dune-project#L1

Reply:
The reason for the difference is the `dune-workspace` in the flow repository uses `3.2` so it was falling some lower bounds CI: https://github.com/facebook/flow/blob/392b6d3008caa40a38627de876f3cd29611f6be4/dune-workspace 
That's why I bumped it to 3.2

Reply:
I see! thanks for the explanation.

(it's probably out of the scope, but IIUC [it is not recommended to check-in dune-workspace](https://dune.readthedocs.io/en/stable/usage.html#:~:text=while%20it%20is%20less%20common%20to%20commit%20dune%2Dworkspace%20files). )